mappings/net/minecraft/world/gen/decorator/TreeDecoratorType.mapping

```diff
@@ -1,2 +1,6 @@
 CLASS net/minecraft/class_4663 net/minecraft/world/gen/decorator/TreeDecoratorType
+	FIELD field_21325 deserializer Ljava/util/function/Function;
+	METHOD method_23472 deserialize (Lcom/mojang/datafixers/Dynamic;)Lnet/minecraft/class_4662;
 	METHOD method_23473 register (Ljava/lang/String;Ljava/util/function/Function;)Lnet/minecraft/class_4663;
+		ARG 0 registerString
+		ARG 1 registerTypeObject
```

Comment:
```suggestion
		ARG 0 id
		ARG 1 deserializer
```
plz

Reply:
@liach  lets come up with something better then 
```
deserializer
``

that functioin is called all over the place and don't want peps to get confused

Reply:
isn't this impl like 
```java
return Registry.register(Registry.TREE_DECORATOR_TYPE, new Identifier(id), new TreeDecoratorType(deserializer));
```
this is the deserializer that converts a datafixerupper dynamic to a tree decorator. or show me what it actually is.

Reply:
Ok Cool, I am so use to not being able to call things things like "id" and "deserializer" or "serializer" ect | I'm old lol

Reply:
Fixed
https://github.com/FabricMC/yarn/pull/1218/commits/4237b9298bebe3682e4dd66f74902ba36958f6f8